github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -266,3 +266,49 @@ periodics:
         resources:
           requests:
             memory: "29Gi"
+- name: periodic-kubevirt-mirror-uploader
+  cluster: ibm-prow-jobs
+  cron: "05 7 * * *"
+  max_concurrency: 1
+  decorate: true
+  decoration_config:
+    timeout: 1h
+    grace_period: 5m
+  extra_refs:
+  - org: fgimenez
+    repo: project-infra
+    base_ref: add-git-pr-script
+    workdir: true
+  - org: kubevirt
+    repo: kubevirt
+    base_ref: master
+  spec:
+    containers:
+      - image: "kubevirt/builder@sha256:e6a76ec71f2de87c3bc648bae87ab43a113540bf57da69f07f76bd0edfd500ab"
+        env:
+        - name: GIT_ASKPASS
+          value: "hack/git-askpass.sh"
+        - name: GOOGLE_APPLICATION_CREDENTIALS
+          value: /etc/gcs/service-account.json
+        command: ["/bin/sh"]
+        args:
+          - "-c"
+          - curl -L https://github.com/bazelbuild/bazelisk/releases/download/v1.7.4/bazelisk-linux-amd64 --output /usr/bin/bazelisk &&
+            chmod a+x /usr/bin/bazelisk &&
+            hack/git-pr.sh -c "bazelisk run //plugins/cmd/uploader:uploader -- -workspace ../../kubevirt/kubevirt/WORKSPACE" -p ../../kubevirt/kubevirt
```

Comment:
Why is `-p` added here? Are you sure that this is needed?` We would also need `-dry-run=false`.

Reply:
Ah never mind regarding the `-p`. I thought for a moment that it was passed to `uploader`, which it is not.

Reply:
yep, added `-dry-run=false`, moved to project-infra periodics and added containerized-data-importer as extra_ref plus a call to execute the uploader on it